slave.mk

```diff
@@ -1292,8 +1293,6 @@ $(addprefix $(TARGET_PATH)/, $(SONIC_INSTALLERS)) : $(TARGET_PATH)/% : \
 		$(if $($(docker:-dbg.gz=.gz)_MACHINE),\
 			mv $($(docker:-dbg.gz=.gz)_CONTAINER_NAME).sh $($(docker:-dbg.gz=.gz)_MACHINE)_$($(docker:-dbg.gz=.gz)_CONTAINER_NAME).sh
 		)
-		$(foreach file, $($(docker)_SHARED_FILES), \
```

Comment:
>$(foreach file, $($(docker)_SHARED_FILES), \ [](http://example.com/codeflow?start=2&length=44)

why removing?

Reply:
We added that originally to store rsyslog_plugin, but with deb pkg now we won't need.